content/docs/guides/components/istio.md

```diff
@@ -82,6 +84,11 @@ ks param set ${MODEL_COMPONENT} injectIstio true
 
 This will inject an istio sidecar in the TF serving deployment.
 
+#### Routing with Istio vs Ambassador
```

Comment:
What's the rationale to allow two types of routing? It's confusing from a users perspective, and multiple ingress paths make creating a secure perimeter more difficult.

Reply:
We want to move to istio, but we can't get rid of ambassador at one time.
So it's a temporary state